manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/mysql.yaml

```diff
@@ -166,9 +166,20 @@ spec:
           args:
           # https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_ignore-db-dir
           # Ext4, Btrfs etc. volumes root directories have a lost+found directory that should not be treated as a database.
-          - --ignore-db-dir=lost+found
+          # ignore-db-dir option has been deprecated in mysql v5.7.16.
+          #
+          # In order to upgrade MySQL to v8.0 remove /var/lib/mysql/lost+found folder in mysql-pv-claim 
+          # (mysql-persistent-storage):
+          #
+          # kubectl exec -it <mysql-pod-name> -n kubeflow -- bash
+          # rm -rf /var/lib/mysql/lost+found
```

Comment:
Chatted offline, we'll see if we can either avoid this or automate it during deployment upgrade.

/hold

Reply:
Fixed